po-docs/ja/virt-list-filesystems.1
.\" Automatically generated by Podwrapper::Man 1.40.2 (Pod::Simple 3.35)
.\"
.\" Standard preamble:
.\" ========================================================================
.de Sp \" Vertical space (when we can't use .PP)
.if t .sp .5v
.if n .sp
..
.de Vb \" Begin verbatim text
.ft CW
.nf
.ne \\$1
..
.de Ve \" End verbatim text
.ft R
.fi
..
.\" Set up some character translations and predefined strings.  \*(-- will
.\" give an unbreakable dash, \*(PI will give pi, \*(L" will give a left
.\" double quote, and \*(R" will give a right double quote.  \*(C+ will
.\" give a nicer C++.  Capital omega is used to do unbreakable dashes and
.\" therefore won't be available.  \*(C` and \*(C' expand to `' in nroff,
.\" nothing in troff, for use with C<>.
.tr \(*W-
.ds C+ C\v'-.1v'\h'-1p'\s-2+\h'-1p'+\s0\v'.1v'\h'-1p'
.ie n \{\
.    ds -- \(*W-
.    ds PI pi
.    if (\n(.H=4u)&(1m=24u) .ds -- \(*W\h'-12u'\(*W\h'-12u'-\" diablo 10 pitch
.    if (\n(.H=4u)&(1m=20u) .ds -- \(*W\h'-12u'\(*W\h'-8u'-\"  diablo 12 pitch
.    ds L" ""
.    ds R" ""
.    ds C` ""
.    ds C' ""
'br\}
.el\{\
.    ds -- \|\(em\|
.    ds PI \(*p
.    ds L" ``
.    ds R" ''
.    ds C`
.    ds C'
'br\}
.\"
.\" Escape single quotes in literal strings from groff's Unicode transform.
.ie \n(.g .ds Aq \(aq
.el       .ds Aq '
.\"
.\" If the F register is >0, we'll generate index entries on stderr for
.\" titles (.TH), headers (.SH), subsections (.SS), items (.Ip), and index
.\" entries marked with X<> in POD.  Of course, you'll have to process the
.\" output yourself in some meaningful fashion.
.\"
.\" Avoid warning from groff about undefined register 'F'.
.de IX
..
.nr rF 0
.if \n(.g .if rF .nr rF 1
.if (\n(rF:(\n(.g==0)) \{\
.    if \nF \{\
.        de IX
.        tm Index:\\$1\t\\n%\t"\\$2"
..
.        if !\nF==2 \{\
.            nr % 0
.            nr F 2
.        \}
.    \}
.\}
.rr rF
.\" ========================================================================
.\"
.IX Title "virt-list-filesystems 1"
.TH virt-list-filesystems 1 "2019-02-07" "libguestfs-1.40.2" "Virtualization Support"
.\" For nroff, turn off justification.  Always turn off hyphenation; it makes
.\" way too many mistakes in technical documents.
.if n .ad l
.nh
.SH "名前"
.IX Header "名前"
virt-list-filesystems \- 仮想マシンまたはディスクイメージのファイルシステムの一覧表示
.SH "書式"
.IX Header "書式"
.Vb 1
\& virt\-list\-filesystems [\-\-options] domname
\&
\& virt\-list\-filesystems [\-\-options] disk.img [disk.img ...]
.Ve
.SH "非推奨"
.IX Header "非推奨"
This tool is obsolete.  Use \fBvirt\-filesystems\fR\|(1) as a more flexible
replacement.
.SH "説明"
.IX Header "説明"
\&\f(CW\*(C`virt\-list\-filesystems\*(C'\fR is a command line tool to list the filesystems that
are contained in a virtual machine or disk image.
.PP
\&\f(CW\*(C`virt\-list\-filesystems\*(C'\fR is just a simple wrapper around \fBlibguestfs\fR\|(3)
functionality.  For more complex cases you should look at the
\&\fBguestfish\fR\|(1) tool.
.SH "オプション"
.IX Header "オプション"
.IP "\fB\-\-help\fR" 4
.IX Item "--help"
簡単なヘルプを表示します。
.IP "\fB\-\-version\fR" 4
.IX Item "--version"
バージョン番号を表示して、終了します。
.IP "\fB\-c \s-1URI\s0\fR" 4
.IX Item "-c URI"
.PD 0
.IP "\fB\-\-connect \s-1URI\s0\fR" 4
.IX Item "--connect URI"
.PD
libvirt を使用していると、指定された \fI\s-1URI\s0\fR に接続します。  省略すると、デフォルトの libvirt ハイパーバイザーに接続します。
.Sp
仮想マシンのブロックデバイスを直接指定すると、libvirt はまったく使用されません。
.IP "\fB\-\-format\fR raw" 4
.IX Item "--format raw"
Specify the format of disk images given on the command line.  If this is
omitted then the format is autodetected from the content of the disk image.
.Sp
If disk images are requested from libvirt, then this program asks libvirt
for this information.  In this case, the value of the format parameter is
ignored.
.Sp
If working with untrusted raw-format guest disk images, you should ensure
the format is always specified.
.IP "\fB\-l\fR" 4
.IX Item "-l"
.PD 0
.IP "\fB\-\-long\fR" 4
.IX Item "--long"
.PD
With this option, \f(CW\*(C`virt\-list\-filesystems\*(C'\fR displays the type of each
filesystem too (where \*(L"type\*(R" means \f(CW\*(C`ext3\*(C'\fR, \f(CW\*(C`xfs\*(C'\fR etc.)
.IP "\fB\-a\fR" 4
.IX Item "-a"
.PD 0
.IP "\fB\-\-all\fR" 4
.IX Item "--all"
.PD
Normally we only show mountable filesystems.  If this option is given then
swap devices are shown too.
.SH "関連項目"
.IX Header "関連項目"
\&\fBguestfs\fR\|(3), \fBguestfish\fR\|(1), \fBvirt\-cat\fR\|(1), \fBvirt\-tar\fR\|(1),
\&\fBvirt\-filesystems\fR\|(1), \fBvirt\-list\-partitions\fR\|(1), \fBSys::Guestfs\fR\|(3),
\&\fBSys::Virt\fR\|(3), http://libguestfs.org/.
.SH "著者"
.IX Header "著者"
Richard W.M. Jones http://people.redhat.com/~rjones/
.SH "COPYRIGHT"
.IX Header "COPYRIGHT"
Copyright (C) 2009 Red Hat Inc.
.SH "LICENSE"
.IX Header "LICENSE"
.SH "BUGS"
.IX Header "BUGS"
To get a list of bugs against libguestfs, use this link:
https://bugzilla.redhat.com/buglist.cgi?component=libguestfs&product=Virtualization+Tools
.PP
To report a new bug against libguestfs, use this link:
https://bugzilla.redhat.com/enter_bug.cgi?component=libguestfs&product=Virtualization+Tools
.PP
When reporting a bug, please supply:
.IP "\(bu" 4
The version of libguestfs.
.IP "\(bu" 4
Where you got libguestfs (eg. which Linux distro, compiled from source, etc)
.IP "\(bu" 4
Describe the bug accurately and give a way to reproduce it.
.IP "\(bu" 4
Run \fBlibguestfs\-test\-tool\fR\|(1) and paste the \fBcomplete, unedited\fR
output into the bug report.


ChangeLog
commit 1f026244ba5891a00617f4ab8c41b8680ff8c497
Author: Richard W.M. Jones <rjones@redhat.com>
Date:   Thu Jan 17 12:06:36 2019 +0000

    Version 1.40.1.

commit ae2aa14527605ffb89d86bf93bf8ca4f969118e1
Author: Richard W.M. Jones <rjones@redhat.com>
Date:   Thu Jan 17 12:03:07 2019 +0000

    valgrind: Add workaround for slow loading of debuginfo by valgrind.
    
    See nbdkit commit
    https://github.com/libguestfs/nbdkit/commit/bd32d899edb4c592a2df0ce4b26a2cf8396eba1d
    and https://bugzilla.redhat.com/show_bug.cgi?id=1662656
    
    (cherry picked from commit ea7c13fed7922b233db8382b1348dfb436e3ba85)

commit 1de72f8f00600bcaa76742dcfd366fb702eef3c2
Author: Richard W.M. Jones <rjones@redhat.com>
Date:   Tue Jan 15 14:39:41 2019 +0000

    Version 1.40.0.

commit 6151d8694d9db0d709a9ad807a5ca6a098d6502c
Author: Richard W.M. Jones <rjones@redhat.com>
Date:   Wed Jan 16 11:49:45 2019 +0000

    po: Drop ca and pt_BR from linguas.
    
    Commit 4252a490bb117940c1d38c838345ed110457386e dropped the PO files
    for ca (Catalan) and pt_BR (Brazilian Portuguese).  However it did not
    drop these languages from the linguas line in the Makefile resulting
    in a build failure.

commit d1ff62f3e08fd8af5e01cff8045f0a01a5aa0597
Author: Richard W.M. Jones <rjones@redhat.com>
Date:   Wed Jan 16 11:39:33 2019 +0000

    configure: Move remaining detection of p2v features to m4/guestfs-v2v.m4.
    
    ./configure output now contains a distinct section for v2v and p2v:
    
      --- Checking the virt-v2v and virt-p2v dependencies ---
      checking for the nbdkit python plugin name... python3
      checking for --with-gtk option... 2
      checking for GTK... yes
      checking for DBUS... yes
      checking if we can build virt-p2v... yes, with Gtk 2
    
    Thanks: Pino Toscano.

commit 61b86bac2e2071c1760b46d13195b4b8e24abd7d
Author: Richard W.M. Jones <rjones@redhat.com>
Date:   Tue Jan 8 12:11:20 2019 +0000

    v2v: -o rhv-upload: Allow configure to set the nbdkit Python version.
    
    No functional change, but it does allow downstream distributions to
    adjust the nbdkit Python plugin used by virt-v2v -o rhv-upload mode:
    
    ./configure --with-virt-v2v-nbdkit-python-plugin=...

commit 090f5589e7b838375bc5333fe0f7972630001348
Author: Richard W.M. Jones <rjones@redhat.com>
Date:   Tue Jan 15 14:43:28 2019 +0000

    po-docs: uk: Fix inconsistent use of \n in translation.
    
    Invalid po file ./uk.po:
    ./uk.po:90428: 'msgid' and 'msgstr' entries do not both begin with '\n'
    msgfmt: found 1 fatal error

commit 1db01404281b2db740f8b854751efc083dcbffa1
Author: Richard W.M. Jones <rjones@redhat.com>
Date:   Tue Jan 15 13:43:29 2019 +0000

    docs: Finalize release notes for 1.40.
    
    Updates commit 0ce8ecdf6627bedfebc235fc4fd69f435a480e88.

commit b8882fed975a09bf7aa14f09eb343ac74c899e71
Author: Richard W.M. Jones <rjones@redhat.com>
Date:   Tue Jan 15 13:42:18 2019 +0000

    website: Update website for 1.40.

commit cf6b527824b2a8dc6e8bc65e38ebdceb227e6db1
Author: Pino Toscano <ptoscano@redhat.com>
Date:   Mon Jan 14 17:07:45 2019 +0100

    inspect: fix inspection of partition-less devices (RHBZ#1661038)
    
    When parsing "xdev"-kind devices, do not assume that the partition
    number can be converted to integer: re_xdev accepts an empty part of the
    partition number, so just handle as it is, as string.
    
    This fixes a regression due to the conversion of the inspection code to
    OCaml, as the old C version did not have this issue.

commit d3ff88c894b0ddadea005b8563b5793fa6b5b9be
Author: Pino Toscano <ptoscano@redhat.com>
Date:   Fri Jan 11 17:00:11 2019 +0100

    OCaml: use the new behaviour of Std_utils.which
    
    Since Std_utils.which can handle relative/absolute paths, remove the
    manual checks, and use Std_utils.which directly.

commit ebe2c7b45832f41e75deb73d784f05ebc87c9b71
Author: Pino Toscano <ptoscano@redhat.com>
Date:   Fri Jan 11 15:59:03 2019 +0100

    mlstdutils: allow relative/absolute paths for Std_utils.which
    
    Make Std_utils.which behave a bit more like which(1), checking the
    existance of relative/absolute paths specified.

commit 88a754bd072d26b660964e9883303a2939d28fc5
Author: Pino Toscano <ptoscano@redhat.com>
Date:   Fri Jan 11 15:19:48 2019 +0100

    mlstdutils: add a very simple test for Std_utils.which

commit 27cd2888c31dc3b93a1c8973010509d69ac38778
Author: Pino Toscano <ptoscano@redhat.com>
Date:   Mon Jan 14 15:18:34 2019 +0100

    automake2junit.ml: string -> bytes
    
    Make it usable OOTB with OCaml >= 4.02.0.

commit 7f97433a3c067ea7c2a2a159b09720c0772605c8
Author: Yuri Chornoivan <yurchor@ukr.net>
Date:   Fri Jan 11 17:14:21 2019 +0000

    Fix miscellaneous typos in comments.

commit 4040c7de93053f93674dac3fc8523feef207fa5f
Author: Richard W.M. Jones <rjones@redhat.com>
Date:   Tue Jan 8 18:29:14 2019 +0000

    Update gnulib to latest version.

commit 4252a490bb117940c1d38c838345ed110457386e
Author: Richard W.M. Jones <rjones@redhat.com>
Date:   Tue Jan 8 16:11:31 2019 +0000

    Update PO files from Zanata.

commit fc028bf57a3ff128d21b904583f9ea02f672ed5b
Author: Richard W.M. Jones <rjones@redhat.com>
Date:   Tue Jan 8 14:03:25 2019 +0000

    v2v: -o openstack: Don't echo full commands (RHBZ#1664310).
    
    They can contain passwords or tokens if for example the
    ‘-oo os-password’ option is used.
    
    Thanks: Tomáš Golembiovský, Brett Thurber.

commit 05d4fcb64d98d4ff1d57560c566ca8e66b695277
Author: Richard W.M. Jones <rjones@redhat.com>
Date:   Tue Jan 8 11:58:30 2019 +0000

    Update copyright dates for 2019.
    
    This command run over the source:
    
    perl -pi.bak -e 's/(20[01][0-9])-2018/$1-2019/g' `git ls-files`

commit 0ce8ecdf6627bedfebc235fc4fd69f435a480e88
Author: Richard W.M. Jones <rjones@redhat.com>
Date:   Mon Jan 7 16:48:15 2019 +0000

    Update release notes for 1.40 release.

commit 492a945791b43f80a769a53e60d0899b3d7c60ab

po-docs/uk/virt-list-filesystems.pl
=encoding utf8

=head1 НАЗВА

virt-list-filesystems - List filesystems in a virtual machine or disk image

=head1 КОРОТКИЙ ОПИС

 virt-list-filesystems [--параметри] -d назва_домену

 virt-list-filesystems [--параметри] образ.img [образ.img ...]

=head1 ЗАСТАРІЛЕ

This tool is obsolete.  Use L<virt-filesystems(1)> as a more flexible
replacement.

=head1 ОПИС

C<virt-list-filesystems> is a command line tool to list the filesystems that
are contained in a virtual machine or disk image.

C<virt-list-filesystems> is just a simple wrapper around L<libguestfs(3)>
functionality.  For more complex cases you should look at the
L<guestfish(1)> tool.

=head1 ПАРАМЕТРИ

=over 4

=item B<--help>

Показати коротку довідку.

=item B<--version>

Показати дані щодо версії і завершити роботу.

=item B<-c адреса>

=item B<--connect адреса>

Якщо використовується libvirt, встановити з’єднання з вказаним I<URI>. Якщо
пропущено, з’єднання буде встановлено з типовим гіпервізором libvirt.

Якщо вказати блокові пристрої гостьових систем безпосередньо, libvirt не
буде використовуватися взагалі.

=item B<--format> raw

Specify the format of disk images given on the command line.  If this is
omitted then the format is autodetected from the content of the disk image.

If disk images are requested from libvirt, then this program asks libvirt
for this information.  In this case, the value of the format parameter is
ignored.

If working with untrusted raw-format guest disk images, you should ensure
the format is always specified.

=item B<-l>

=item B<--long>

With this option, C<virt-list-filesystems> displays the type of each
filesystem too (where "type" means C<ext3>, C<xfs> etc.)

=item B<-a>

=item B<--all>

Normally we only show mountable filesystems.  If this option is given then
swap devices are shown too.

=back

=head1 ЕКРАНУВАННЯ СИМВОЛІВ У ОБОЛОНЦІ

Libvirt guest names can contain arbitrary characters, some of which have
meaning to the shell such as C<#> and space.  You may need to quote or
escape these characters on the command line.  See the shell manual page
L<sh(1)> for details.

=head1 ТАКОЖ ПЕРЕГЛЯНЬТЕ

L<guestfs(3)>, L<guestfish(1)>, L<virt-cat(1)>, L<virt-tar(1)>,
L<virt-filesystems(1)>, L<virt-list-partitions(1)>, L<Sys::Guestfs(3)>,
L<Sys::Virt(3)>, L<http://libguestfs.org/>.

=head1 АВТОР

Richard W.M. Jones L<http://people.redhat.com/~rjones/>

=head1 АВТОРСЬКІ ПРАВА

© Red Hat Inc., 2009

